<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="rupesh" failures="1" tests="1" name="AddReportGroup_Test" time="28.213" errors="0" timestamp="22 Apr 2020 07:45:30 GMT">
  <testcase classname="com.dataseers.qa.testcases.AddReportGroup_Test" name="Add_NewReport_Test" time="22.027">
    <failure type="java.lang.AssertionError" message="Save new report scenario failed">
      <![CDATA[java.lang.AssertionError: Save new report scenario failed
	at com.dataseers.qa.pages.administration.ReportGroups.savereport(ReportGroups.java:165)
	at com.dataseers.qa.testcases.AddReportGroup_Test.Add_NewReport_Test(AddReportGroup_Test.java:58)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Add_NewReport_Test -->
</testsuite> <!-- AddReportGroup_Test -->
